v22.1/follower-reads.md

```diff
@@ -1,197 +1,191 @@
 
-To verify that your cluster is performing follower reads, go to the [Custom Chart Debug Page in the DB Console](ui-custom-chart-debug-page.html) and add the metric `follower_read.success_count` to the time-series graph you see there. The number of follower reads performed by your cluster will be shown.
+To set `AS OF SYSTEM TIME follower_read_timestamp()` on all implicit and explicit read-only transactions by default, use one of the following options:
 
-### Run queries that use exact staleness follower reads
+- Set the `default_transaction_use_follower_reads` [session variable](set-vars.html) to `on`. When `default_transaction_use_follower_reads=on`, all read-only transactions use exact staleness follower reads.
+- Execute the `SET SESSION CHARACTERISTICS AS TRANSACTION AS OF SYSTEM TIME follower_read_timestamp()` [SQL statement](set-vars.html#special-syntax-cases). This has the same effect as setting the session variable as shown above.
 
-Any [`SELECT` statement](select-clause.html) with an appropriate [`AS OF SYSTEM TIME`](as-of-system-time.html) value can be an exact staleness follower read.
+You can set `default_transaction_use_follower_reads` on a per-role basis; for instructions, see [Set default session variable values for a role](alter-role.html#set-default-session-variable-values-for-a-role).
+{{site.data.alerts.end}}
 
-For exact staleness reads, you can use the convenience [function](functions-and-operators.html) `follower_read_timestamp()`, which returns a [`TIMESTAMP`](timestamp.html) that provides a high probability of being served locally while not [blocking on conflicting writes](#exact-staleness-reads-and-long-running-writes).
+### Bounded staleness reads
 
-Use this function in an `AS OF SYSTEM TIME` statement as follows:
+A _bounded staleness read_ is a historical read that uses a dynamic, system-determined timestamp to minimize staleness while being more tolerant to replication lag than an exact staleness read. Bounded staleness reads also help increase system availability, since they provide the ability to serve reads from local replicas even in the presence of network partitions or other failures that prevent the SQL gateway from communicating with the leaseholder.
 
-``` sql
-SELECT ... FROM ... AS OF SYSTEM TIME follower_read_timestamp();
-```
+#### When to use bounded staleness reads
 
-### Run queries that use bounded staleness follower reads
+Use bounded staleness follower reads when you:
 
-As described in [Bounded staleness read limitations](#bounded-staleness-read-limitations), bounded staleness follower reads are available under the following conditions:
+- Need minimally stale reads from the nearest replica without blocking on conflicting transactions. This is possible because the historical timestamp is chosen dynamically and the least stale timestamp that can be served locally without blocking is used.
```

Comment:
"conflicting transactions" (here and elsewhere, if used) could link to https://www.cockroachlabs.com/docs/v21.2/transactions.html#transaction-contention so users can read more about this if needed